src/components/MemberSidebar/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Icon, Image } from 'semantic-ui-react';
+
+import {
+  SidebarWrapper,
+  SidebarElement,
+  SidebarElementSelected,
+  SidebarIconArea,
+  SidebarIcon,
+  SidebarUserName
+} from 'components/Shared/Sidebar/styles';
+
+class MemberSidebar extends Component {
+  render() {
+    return (
+      <SidebarWrapper>
+        <SidebarIconArea>
+          <SidebarIcon>
+            <Image
+              src="https://react.semantic-ui.com/images/wireframe/square-image.png"
+              circular
+            />
+            <SidebarUserName>Familyname Firstname</SidebarUserName>
+          </SidebarIcon>
+        </SidebarIconArea>
+
+        <SidebarElementSelected>
```

Comment:
I think you can refactor it. Eg:
const Element = ({icon, text, isSelected = false}) => {
   const El = isSelected ? SidebarElementSelected : SidebarElement
   return (
     <El>
        <p>
          <Icon name={icon} /> {text}
        </p>
     </El>
   )
}

const items = [
  {
     icon: "copy outline",
     text: "Yout Daily Reports"
  }
  ...
]

Then map it on render

{items.map((item) => <Element icon={item.icon} text={item.text} />)}